beater/connbeat.go

```diff
@@ -76,6 +77,9 @@ func toMap(containerInfo *ContainerInfo) common.MapStr {
 			"id":        containerInfo.id,
 			"local_ips": containerInfo.localIPs.ToSlice(),
 			"env":       containerInfo.environment,
+			"docker_host": common.MapStr{
+				"name": containerInfo.hostName,
+			},
```

Comment:
I would prefer to rename "docker_host" to "container_host" or "parent_host" to make it more general. I also like to rename "name" to "hostname" to clarify that the information residing there is the hostname of the machine and not some other name.